homeassistant/components/sensibo/config_flow.py

```diff
@@ -0,0 +1,99 @@
+"""Adds config flow for Sensibo integration."""
+from __future__ import annotations
+
+import asyncio
+import logging
+
+import aiohttp
+import async_timeout
+from pysensibo import SensiboClient, SensiboError
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_API_KEY, CONF_NAME
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+import homeassistant.helpers.config_validation as cv
+
+from .const import _INITIAL_FETCH_FIELDS, DEFAULT_NAME, DOMAIN, TIMEOUT
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_API_KEY): cv.string,
+        vol.Required(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    }
+)
+
+
+class SensiboConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Sensibo integration."""
+
+    VERSION = 1
+
+    entry: config_entries.ConfigEntry
+
+    def __init__(self) -> None:
+        """Initialize."""
+        self.name: str = ""
+        self.api_key: str = ""
+        self.id: list = []
+        self.errors: dict[str, str] = {}
```

Comment:
Don't store errors. This gives the impression that errors can be passed between steps which is not allowed